MeshLib/PropertyVector.h

```diff
@@ -155,13 +156,31 @@ friend class Properties;
 
     void initPropertyValue(std::size_t group_id, T const& value)
     {
-        _values[group_id] = new T(value);
+        if (_n_components != 1)
+            OGS_FATAL("Single-component version of initPropertyValue() is called "
+                      "for a multi-components PropertyVector<T*>");
+        T* p = new T[1];
+        p[0] = value;
```

Comment:
Note: This will not work if `T` is an abstract class or `T` is not copyable. However, it did not work either before.